workspaces/factory-workspace/config/99-local.conf

```diff
@@ -0,0 +1,3 @@
+ALL_DEBUG = D_FULLDEBUG, D_SECURITY
```

Comment:
These are the 2 files in shared/config.  I think this one is not even used

Reply:
Removed the file.